dev/generate-tests.py

```diff
@@ -1220,7 +1220,18 @@ def gencudaunittests(specdict):
                         for arg, val in test["inputs"].items():
                             typename = gettype(arg, spec.args)
                             if "List" not in typename:
-                                f.write(" " * 4 + arg + " = " + str(val) + "\n")
+                                if arg == "identity":
+                                    f.write(
+                                        " " * 4
+                                        + arg
+                                        + " = "
+                                        + str(dtypes[0])
+                                        + "("
+                                        + str(val)
+                                        + ")\n"
+                                    )
+                                else:
+                                    f.write(" " * 4 + arg + " = " + str(val) + "\n")
```

Comment:
Maybe f-strings would be cleaner?
```suggestion
                                    f.write(f"    {arg} = {dtypes[0]}({val})\n")
                                else:
                                    f.write(f"    {arg} = {val}\n")
```